core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -888,7 +888,7 @@ object KafkaConfig {
   val ReplicaLagTimeMaxMsDoc = "If a follower hasn't sent any fetch requests or hasn't consumed up to the leaders log end offset for at least this time," +
   " the leader will remove the follower from isr"
   val ReplicaSocketTimeoutMsDoc = "The socket timeout for network requests. Its value should be at least replica.fetch.wait.max.ms"
-  val ReplicaSocketReceiveBufferBytesDoc = "The socket receive buffer for network requests"
+  val ReplicaSocketReceiveBufferBytesDoc = "The socket receive buffer for network requests to the leader for replicating data"
```

Comment:
I copy the description from our 0.8.2 doc (https://svn.apache.org/repos/asf/kafka/site/082/configuration.html#topic-config)